ci-operator/config/konveyor/pelorus/konveyor-pelorus-master.yaml

```diff
@@ -0,0 +1,65 @@
+base_images:
+  cli:
+    name: "4.9"
+    namespace: ocp
+    tag: cli
+  ubi_minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+binary_build_commands: make dev-env
+build_root:
+  project_image:
+    dockerfile_literal: |
+      FROM registry.access.redhat.com/ubi8
+      USER root
+      RUN yum install -y python39 make diffutils openssl git && \
+          yum clean all && rm -rf /var/cache/dnf/*
+      RUN git clone https://github.com/konveyor/pelorus.git
```

Comment:
I don't think you need that as you should have access to the source code from the context, in short you already should have the source code here. What could be beneficial in this place is actually building of exporter images. I would do this later.

Reply:
And if you do need to clone it here then maybe --depth 1